SECURITY.md

```diff
@@ -0,0 +1,59 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, either:
+
+1. Report it on Github directly you can follow the procedure described
+[here](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)
```

Comment:
This Github provided process to private report a vulnerability and collaborate on the fix privately is quite nice.  I don't think this was available for us in the past when creating security policies for other projects.  Thanks for finding this, it does look like a nice workflow!!

Reply:
Thanks to @AdamKorcz for pointing that out!